gajosu/ha-whatsapp-web-rest-api
# Remove deprecated POST endpoint `/api/messages

This pull request removes the deprecated POST endpoint `/api/messages`, as it has been replaced with the updated endpoint `/api/chats/:id/messages`.

### Changes Made

- Removed the deprecated POST endpoint `/api/messages`.
- Updated the documentation and code comments to reflect the removal of the deprecated endpoint.
- Updated any relevant references or links pointing to the deprecated endpoint.

### How to Test

1. Start the application.
2. Attempt to make a POST request to the deprecated endpoint `/api/messages`.
3. Verify that the request returns a "404 Not Found" response, indicating that the endpoint has been removed.
4. Make a POST request to the updated endpoint `/api/chats/:id/messages` with the appropriate chat ID to ensure that the new endpoint functions as expected.

### Checklist

- [x] Removed the deprecated POST endpoint `/api/messages`.
- [x] Updated the documentation and comments accordingly.
- [x] Verified the removal by testing the deprecated endpoint and confirming it returns a "404 Not Found" response.
